app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -8,6 +8,12 @@
     text-align: start;
   }
 
+  &.rich-govspeak {
+    strong {
+      font-weight: bold;
```

Comment:
Seems to be true with Manuals too:
https://github.com/alphagov/manuals-frontend/blob/master/app/assets/stylesheets/_reset.scss

So in those cases these rules would work, but perhaps not everywhere.